modernizr.config.json

```diff
@@ -1,30 +0,0 @@
-{
```

Comment:
Is it safe to remove this and also .yarnrc and .broweserslistrc?

Reply:
I think it's fine to remove the .yarnrc. As for the others, The usage I found for them was when building the website, so I removed them.

If I missed something and the files are still needed, let me know.